### PR TITLE
Feat(modern auth): new mobile auth feature

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Comment by plondon** [comment]
@milan-bc merge conflicts and a pretty crazy diff

**Comment by milan-bc** [comment]
@plondon I see now, weird, did you maybe updated `secure-channel-login` with latest `development`?

File: packages/blockchain-wallet-v4-frontend/src/components/DropdownLanguage/index.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ class DropdownLanguageContainer extends React.PureComponent {
 
     return (
       <SimpleDropdown
-        color='whiteFade900'
+        color={color || 'gray400'}
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/components/DropdownLanguage/index.js:27]: this component have prop `color` so we should use it

File: packages/blockchain-info-components/src/Images/Images.ts [hunk 2]
```diff
@@ -124,6 +125,7 @@ const Images = {
   'allow-camera2': allowCamera2,
   'allow-camera3': allowCamera3,
   'apple-app-store-badge': appleAppStoreBadge,
+  'bg-illo': bgIllo,
```
Comment (plondon) [inline packages/blockchain-info-components/src/Images/Images.ts:128]: Illo?

**Comment by plondon** [comment]
Screen Shot 2020-09-23 at 9 08 12 AM
blockchain.com logo offcenter left 24px

**Comment by plondon** [comment]
Screen Shot 2020-09-23 at 9 09 11 AM

mobile

**Comment by plondon** [comment]
footer:
Language Selection | Version | Need help?

does not match designs

**Comment by plondon** [comment]
Screen Shot 2020-09-23 at 9 10 55 AM

qr code logo not centered

**Comment by plondon** [comment]
I can now scroll left and right on the login page

https://gyazo.com/8d05a3ef1ec6b69fecd0a587b62d582c

File: packages/blockchain-info-components/src/Images/Images.ts (hunk 2 above)
Comment (milan-bc) [reply packages/blockchain-info-components/src/Images/Images.ts:128]: that is how design call it I just reuse name, but we can rename it to some bg-pattern or something.

File: packages/blockchain-wallet-v4-frontend/src/layouts/Public/index.js [hunk 3]
```diff
@@ -49,9 +49,8 @@ const ContentContainer = styled.div`
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  padding: 0 32px;
   z-index: 1;
-  max-width: 100%;
+  width: 100%;
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/layouts/Public/index.js:53]: prevent scroll fix cc @plondon

**Pull request merged**